### ISSUE TITLE
Monthly PyLadies Meeting Notes: 10th June 2021 @ 6pm UTC

### ISSUE THREAD

**Issue by Marie-L** [pyladies/global-organizing#116]
_This template should be used (edited) the month prior and during the meeting. This will ensure we can address all topics added throughout the month; as well as provide minutes and follow up after the meeting is held._

# PyLadies Global Meeting

- Date: 10th June 
- Previous meeting issue link:
- Current meeting attendees:

## Agenda

_Please add items as needed. If you add a topic please make sure to attend the meeting._

### Project Updates
1. Project Comms (Slack: #project-communication / Repo: https://github.com/pyladies/project-communications)

2. Project Admin & Logistics (Slack: #project-admin-logistics / Repo: https://github.com/pyladies/project-admin-logisitics )

3. Project Tech & Infra (Slack: #project-tech-infra / Repo: https://github.com/pyladies/pyladies )

### Review Actions Items from Last Meeting

Action Item| Owner | Relevant links | Completed
| --| --| --| --|

### New Business

- [ ]
- [ ]
- [ ]
- [ ]
- [ ]

### Old Business: Follow up items from prior meeting:
  - [ ]
  - [ ]
  - [ ]
  - [ ]

### Action Items and Assignees for Next Meeting

Action Item| Owner | Relevant links | 
| --| --| --|

@pyladies/pyladies-global-contributors 
@pyladies/pyladies-global-admin

**Comment by Mariatta** [comment]
Saw the comment about bevy and contract. From my perspective, seems fair to have a contract. Is there something specific in the contract itself that is problematic? Is this something we should discuss with the PSF?

**Comment by BethanyG** [comment]
@Mariatta  - We're not at a contract stage with Bevy (yet).  They were going to meet with the PSF, since they were previously discussing  possibly powering PyCon and other conferences and activities, and then circle back with PyLadies around possibly powering community sites/ events pages.

**Comment by deboraazevedo** [comment]
Hi, everyone! Why is the election on hold? I remember by this time last year we were working on the election documents, the process was voted and decided together with the communities (https://github.com/pyladies/global-organizing/issues/32) and I'd like to know how things are going to proceed, since some terms (the ones with 1 year commitment) finish now

**Comment by lorenanicole** [comment]
It would only be for appointed seats as elected seats are a multi year appointment. As I can tell from reading the issues it sounds like this may be due to the group having just started and wanting more time to continue to work?

**Comment by Marie-L** [comment]
Thanks @lorenanicole 😊 ! @deboraazevedo yes, it's only on hold for the appointed seats. It has been a difficult year due to the pandemic and as a team we have agreed that it would be best to lay a solid foundation before we can introduce new members. If you would like to have a chat about this, please get in touch. Thanks !
